.github/workflows/coverage.yml

```
@@ -36,9 +36,8 @@ env:
 
 jobs:
   coverage:
-    name: Coverage nightly
+    name: Coverage (+stable)
```

Nitpick: can we match the other job names that mention the Rust version?

(Or we could just say "Coverage".)

```suggestion
    name: Coverage on stable
```

---

Old habits :D

---

Sorry for damaging those habits 😆